Source/Fuse.Controls.DatePicker/Android/DatePicker.uno

```diff
@@ -0,0 +1,235 @@
+using Uno;
+using Uno.UX;
+using Uno.Compiler.ExportTargetInterop;
+
+using Fuse;
+using Fuse.Controls;
+using Fuse.Controls.Native;
+
+namespace Fuse.Controls.Native.Android
+{
+	extern(Android) static class DateTimeConverterHelpers
+	{
+		const long DotNetTicksInMs = 10000L;
+		const long UnixEpochInDotNetTicks = 621355968000000000L;
+
+		public static DateTime ConvertMsSince1970InUtcToDateTime(long msSince1970InUtc)
+		{
+			var dotNetTicksRelativeToUnixEpoch = msSince1970InUtc * DotNetTicksInMs;
+			var dotNetTicks = dotNetTicksRelativeToUnixEpoch + UnixEpochInDotNetTicks;
+
+			return new DateTime(dotNetTicks, DateTimeKind.Utc);
+		}
+
+		public static long ConvertDateTimeToMsSince1970InUtc(DateTime dt)
+		{
+			dt = dt.ToUniversalTime();
+
+			var dotNetTicks = dt.Ticks;
```

Comment:
micro-nit, this could have been written like this
```uno
var dotNetTicks = dt.ToUniversalTime().Ticks;
```
...to avoid the temporary. But it's IMO not worth fixing ATM.